templates/config_extra.conf.j2

```diff
@@ -0,0 +1,10 @@
+# Managed by Ansible
```

Comment:
can you remove this file altogether (and the associated task)? It doesn't really make sense to use it for agent 6